project-base/app/tests/FrontendApiBundle/Functional/Login/LoginTest.php

```diff
@@ -16,12 +16,60 @@ class LoginTest extends GraphQlTestCase
      */
     protected TokenFacade $tokenFacade;
 
+    public function testAccessTokenIsNotValidAfterPasswordChange(): void
```

Comment:
Personally, I would expect this test in a new class (what about something like `LoginTokenInvalidationTest`), not in `LoginTest`, however, this is really marginal :shrug: